iam/src/index.ts

```diff
@@ -48,6 +49,7 @@ import {
   TwitterFollowerGTE1000Provider,
   TwitterFollowerGT5000Provider,
   TwitterTweetGT10Provider,
+  Ens,
```

Comment:
This current import has issues. Also the procedureRouter import should be changed to the following 
import { router as procedureRouter } from "@gitcoin/passport-platforms/dist/commonjs/src/procedure-router";